tests/playbooks/tests_ipv6_dns_search.yml

```diff
@@ -0,0 +1,93 @@
+---
+- hosts: all
+  tasks:
+    - import_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: br-example
+            type: bridge
+            ip:
+              dhcp4: false
+              auto6: false
+              dns: 8.8.8.8
+              dns_search: google.com
+              address:
+                - 10.2.2.77/26
+            state: up
+    - block:
+        - name: Verify nmcli connection DNS search entry for IPv4
+          shell: |
+            set -euxo pipefail
+            nmcli -f ipv4.dns-search connection show br-example
+          register: ipv4_dns_search
+          ignore_errors: yes
+          changed_when: false
+
+        - name: Verify nmcli connection DNS search entry for IPv6
+          shell: |
+            set -euxo pipefail
+            nmcli -f ipv6.dns-search connection show br-example
+          register: ipv6_dns_search
+          ignore_errors: yes
+          changed_when: false
+
+        - name: Assert that ipv4 DNS search entry is configured correctly
+          assert:
+            that:
+              - "'google.com' in ipv4_dns_search.stdout"
+            msg: "ipv4 DNS search entry is configured incorrectly"
+
+        - name: Assert that ipv6 DNS search is ignored when ipv6 method
+                is ignore
+          assert:
+            that:
+              - "'google.com' not in ipv6_dns_search.stdout"
+            msg: "ipv6 DNS search is not ignored when ipv6 method is ignore"
+        - name: Reconfigure connection profile and make ipv6 method 'manual'
+          import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: br-example
+                type: bridge
+                ip:
+                  dhcp4: false
+                  auto6: false
+                  dns:
+                    - 8.8.8.8
+                    - 2001:db8::20
+                  dns_search: google.com
+                  address:
+                    - 10.2.2.77/26
```

Comment:
Please also test the case that only static IPv6 is configured.

Reply:
The test case was added.